source/docs/03-metadata.xml

```diff
@@ -1625,6 +1625,43 @@
                               <specDesc key="handList"/>
                            </specList>
                         </p>
+                        <p>
+                           <specList>
+                              <specDesc key="addDesc"/>
+                           </specList>
+                        </p>
+                        <p>In manuscripts there can be various kinds of entries, additions, corrections, marginalia and revisions; all these interventions in the “original” manuscript can be documented under <gi scheme="MEI">addDesc</gi>. However, it is important to understand that these are not additions to the musical text directly. These additions to the manuscript can come from the composer himself, copyists, typesetters, publishers, previous owners, or librarians. These entries are usually indicated in the continuous text with an indication of the location within the manuscript as well as the means of writing used. E.g. “Auf fol. 109v links mit Bleistift von Schindler „NB (Sch.) Hier fehlen 8 Takte (auch im Chor). jedoch die eine spätere […]“” (see <ref target="https://digital.staatsbibliothek-berlin.de/werkansicht?PPN=PPN756658373&amp;PHYSID=PHYS_0398&amp;DMDID=DMDLOG_0005">facsimile</ref>).
```

Comment:
```suggestion
                        <p>In documents (handwritten or printed) there can be various kinds of entries, additions, corrections, marginalia and revisions; all these interventions in the “original” manuscript can be documented under <gi scheme="MEI">addDesc</gi>. However, it is important to understand that these are not additions to the musical text directly. These additions to the document can come from the composer himself, copyists, typesetters, publishers, previous owners, or librarians. These entries are usually indicated in the continuous text with an indication of the location within the document as well as the means of writing used. E.g. “Auf fol. 109v links mit Bleistift von Schindler „NB (Sch.) Hier fehlen 8 Takte (auch im Chor). jedoch die eine spätere […]“” (see <ref target="https://digital.staatsbibliothek-berlin.de/werkansicht?PPN=PPN756658373&amp;PHYSID=PHYS_0398&amp;DMDID=DMDLOG_0005">facsimile</ref>).
```

Reply:
as @ahankinson pointed out (https://github.com/music-encoding/music-encoding/pull/1094#discussion_r1154390361), this can also describe prints, not only manuscripts. I tried to make this paragraph more generic. @gucl-mu do my changes still keep your intention?